lib/kubernetes-deploy/runner.rb

```diff
@@ -55,11 +55,12 @@ def self.with_friendly_errors
       exit 1
     end
 
-    def initialize(namespace:, environment:, current_sha:, context:, wait_for_completion:, template_folder: nil)
+    def initialize(namespace:, current_sha:, context:, wait_for_completion:, template_dir:)
       @namespace = namespace
       @context = context
       @current_sha = current_sha
-      @template_path = File.expand_path('./' + (template_folder || "config/deploy/#{environment}"))
+
+      @template_dir = Pathname.new(template_dir).realpath
```

Comment:
@KnVerey this way we can accept both absolutely and relative paths; relative path will be converted to absolute.